source/installation-guide/installing-wazuh-agent/deployment_variables/deployment_variables_macos.rst

```diff
@@ -47,41 +47,41 @@ Examples:
 .. code-block:: console
 
      # launchctl setenv WAZUH_MANAGER="10.0.0.2" WAZUH_REGISTRATION_PASSWORD="TopSecret" \
-          WAZUH_AGENT_NAME="aix-agent" && installer -pkg wazuh-agent-3.10.1-1.pkg -target
+          WAZUH_AGENT_NAME="aix-agent" && installer -pkg wazuh-agent-3.10.1-1.pkg -target /
```

Comment:
The agent name must be `macos-agent`. Please change all macOS examples and review the consistency.

Reply:
https://github.com/wazuh/wazuh-documentation/pull/1847/commits/013448d95bbccb343c44a17ddeebffb5c620f6e6